packages/python/plotly/plotly/tests/test_core/test_update_objects/test_update_annotations.py

```diff
@@ -219,6 +219,10 @@ def test_update_annotations(self):
             "annotations", [4], patch=dict(showarrow=False), secondary_y=True
         )
 
+    def test_annotation_attributes(self):
+        self.fig.add_annotation(test="this text", yref="paper")
```

Comment:
I think you wanted `text="this text"` here. (CI fails because `test` is not valid)

Reply:
ohh yep!